x/leverage/client/cli/tx.go

```diff
@@ -198,3 +200,69 @@ func GetCmdRepayAsset() *cobra.Command {
 
 	return cmd
 }
+
+// NewCmdSubmitUpdateRegistryProposal returns a CLI command handler to generate
+// or broadcast a transaction with a governance proposal message containing a
+// UpdateRegistryProposal.
+//
+// NOTE: The "registry" provided in the proposal replaces the entire existing
+// registry.
+func NewCmdSubmitUpdateRegistryProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-registry [proposal-file] [deposit]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Submit a update leverage registry governance proposal",
+		Long: strings.TrimSpace(
+			`Submit a leverage registry governance proposal along with an initial deposit.
+The proposal details must be supplied via a JSON file. Please see the UpdateRegistryProposal
+type for a complete description of the expected input.
+
+Example:
+$ umeed tx gov submit-proposal update-registry /path/to/proposal.json [flags...]
+
+Where proposal.json contains:
+
+{
+  "title": "Update the Leverage Token Registry",
+  "description": "Replace the supported tokens in the leverage registry.",
+  "registry": [
+		{
+			"base_denom": "uumee",
+			"reserve_factor": "0.05",
+			"collateral_weight: "0.05",
```

Comment:
@toteki can you suggest reasonable dummy values to use here?

Reply:
These should be fine - maybe increase reserve factor to 0.1 so they're different, but they're in line with values we'd actually use when starting out so no change is really needed.